src/components/Navigation/NavButton.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components/macro'
+import { FaBars } from 'react-icons/fa'
+import { AiOutlineClose } from 'react-icons/ai'
+
+function NavButton({ showNavigation, navigation }) {
+  return (
+    <Button
+      onClick={() => {
+        showNavigation()
+      }}
+    >
+      {navigation ? <AiOutlineClose /> : <FaBars />}
```

Comment:
Are the icons accessible? 🤔 if not, it would be a shame if people with a screenreader don't know what you are trying to tell them. And you should add an `aria-label` to the button.